docker/centos7/Dockerfile

```diff
@@ -128,8 +130,25 @@ RUN source /opt/rh/devtoolset-8/enable && \
         -G Ninja \
         -DLLVM_INCLUDE_EXAMPLES=OFF \
         -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;lld;lldb" \
+        -DLLVM_STATIC_LINK_CXX_STDLIB=ON \
+        ../llvm && \
+    cmake --build . && \
```

Comment:
I recommend to do `mkdir build-clang && cd build-clang` here.
Running `cmake` inside the source directory is not a great practice.

Reply:
line 127 create a sub directory inside the source directory, so I guess I don't need to create another directory, I just need to change its name, right?

Reply:
Oops, sorry, yes, that's right, no change needed. (Sorry, I missed that line as github diff was hiding that line...)